ipfs-cluster-ctl/formatters.go

```diff
@@ -82,23 +83,41 @@ func textFormatPrintIDSerial(obj *api.IDSerial) {
 	}
 
 	fmt.Printf("%s | %d peers\n", obj.ID, len(obj.ClusterPeers))
-	fmt.Println("  > Addresses:")
+	addrs := sort.StringSlice{}
```

Comment:
Hm, that link is acting funny for me, I'm meaning to link to line 351 (not 340).